### PR TITLE
Changes to 3DCursor javadocs

### PR DESCRIPTION
**Pull request by parthmehta209**

### CODE REVIEW

**Comment by liaxim** [comment]
Looks good. Will merge in the afternoon.

File: GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java [hunk 1]
```diff
@@ -747,14 +758,28 @@ public boolean addSelectableObject(GVRSceneObject object) {
         return true;
     }
 
+    /**
+     * This call is for objects for which {@link CursorManager#addSelectableObject(GVRSceneObject)}
+     * was called. After calling this on a {@link GVRSceneObject} there will be no
+     * {@link CursorEvent}s generated when a {@link Cursor} interracts with this
```
Comment (rahul27) [inline GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java]: typo : interacts

File: GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java [hunk 2]
```diff
@@ -815,6 +840,10 @@ public void setSoundEnabled(boolean enable) {
         globalSettings.setSoundEnabled(enable);
     }
 
+    /**
+     * Gets the status of the Sounds for the {@link CursorManager}
+     * @return <code>true</code> is sound is enabled, <code>false</code> if it is disabled
```
Comment (rahul27) [inline GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java]: <code>true</code> **if** sound is enabled,

File: GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/MovableBehavior.java [hunk 3]
```diff
@@ -216,6 +216,13 @@ private void computeRotation(Vector3f start, Vector3f end) {
         rotation.set(cross.x, cross.y, cross.z, real_part).normalize();
     }
 
+    /**
+     *  Returns a unique long value Associated with the {@link MovableBehavior} class. Each
```
Comment (rahul27) [inline GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/MovableBehavior.java]: why is Associated capitalized?

File: GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java (hunk 1 above)
Comment (parthmehta209) [inline GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java]: Done

File: GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java (hunk 2 above)
Comment (parthmehta209) [inline GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/CursorManager.java]: Done

File: GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/MovableBehavior.java (hunk 3 above)
Comment (parthmehta209) [inline GVRf/Extensions/3DCursor/3DCursorLibrary/src/main/java/org/gearvrf/io/cursor3d/MovableBehavior.java]: fixed

**Comment by rahul27** [comment]
Looks good. Can be merged.

**Pull request merged**